presidio-analyzer/Dockerfile

```diff
@@ -1,23 +1,23 @@
 ARG REGISTRY=presidio.azurecr.io
-ARG PRESIDIO_DEPS_LABEL=latest	
+ARG PRESIDIO_DEPS_LABEL=latest
 
-FROM ${REGISTRY}/presidio-python-deps:${PRESIDIO_DEPS_LABEL}	
+FROM ${REGISTRY}/presidio-python-deps:${PRESIDIO_DEPS_LABEL}
 
 ARG NAME=presidio-analyzer
 WORKDIR /usr/bin/${NAME}
 ADD ./${NAME} /usr/bin/${NAME}
 
-RUN pipenv install --dev --sequential && \
+RUN pipenv sync --dev --sequential && \
```

Comment:
I remember 'sync' didnt work when we used it before...

Reply:
That's what we do on the base image